qt/widgets/instrumentview/src/InstrumentWidget.cpp

```diff
@@ -23,6 +23,7 @@
 #include "MantidAPI/IPeaksWorkspace.h"
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/Workspace.h"
+#include "MantidDataObjects/EventWorkspace.h"
```

Comment:
Now that you don't need the complete type, you can change this to the forward declaration: `EventWorkspace_fwd.h`.

Reply:
Actually, you can remove it altogether.